ui/src/main/resources/PhenoTips/MendelianSearch.xml

```diff
@@ -483,13 +483,9 @@ div.mendelian-search-symptoms {
 .variant-categories {
```

Comment:
Talked to Brittney and Veronika about this, we tried out a few different things and thought that this looked best. I'm open to changing it if anyone has strong opinions, will probably ask everyone their opinion on UI at some dev meeting once mendelian-search/variant-store is wrapping up.